contracts/services/osmosis-cl-withdrawer/src/bin/schema/valence-osmosis-cl-lper.json

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
We should ignore schema files

Reply:
oh I didn't realize that we're ignoring everything, I thought we agreed to ignore the raw schemas but keep the top level one. how will front end pull the jsons?

Reply:
Yeah I kinda agree to not generate anything and if someone needs them can just generate them, it's very fast anyways.
This way we are 100% sure they are not using some old schema that we forgot to update at some point

Reply:
Sure, as long as we have a CI to fail when schemas are not accurate I'm actually fine to add them. I just want to avoid the possibility of having outdated schemas for a specific commit

Reply:
Share same concern as @keyleu here, I just don't want to have outdated schemas because we forgot, if we can automate it, im all up for it! 

Also FYI, @elenamik  is taking the schemas from the repo directly, so we need this automated.

FYI2, as far as I can tell, other project (astroport for example) doesn't include schemas in the contracts repo, they might have it on their front end repo tho.

The hard part is not generating the schema, the annoying part is to have to clone the repo to do it, we have a script that does it.

Reply:
ill automate the schema validation checks in CI, np. gonna remove my schema for now then.

astroport does have it, just in a [dedicated directory](https://github.com/astroport-fi/astroport-core/tree/main/schemas). guessing that's to ease the frontenders life as well.